bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoDimmerHandler.java

```diff
@@ -589,21 +617,26 @@ public void setBinaryState(String action, String argument, String value) {
     }
 
     public void setTimerStart(String action, String argument, String value) {
+        if (host.isEmpty()) {
+            logger.error("Failed to set timerStart for device '{}': IP address missing", getThing().getUID());
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+            return;
+        }
+        String wemoURL = getWemoURL(host, BASICEVENT);
+        if (wemoURL == null) {
+            logger.error("Failed to set timerStart for device '{}': URL cannot be created", getThing().getUID());
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+            return;
+        }
         try {
             String soapHeader = "\"urn:Belkin:service:basicevent:1#SetBinaryState\"";
             String content = "<?xml version=\"1.0\"?>"
                     + "<s:Envelope xmlns:s=\"http://schemas.xmlsoap.org/soap/envelope/\" s:encodingStyle=\"http://schemas.xmlsoap.org/soap/encoding/\">"
                     + "<s:Body>" + "<u:SetBinaryState xmlns:u=\"urn:Belkin:service:basicevent:1\">" + value
                     + "</u:SetBinaryState>" + "</s:Body>" + "</s:Envelope>";
-
-            URL descriptorURL = service.getDescriptorURL(this);
-            String wemoURL = getWemoURL(descriptorURL, "basicevent");
-
-            if (wemoURL != null) {
-                wemoCall.executeCall(wemoURL, soapHeader, content);
-            }
+            wemoCall.executeCall(wemoURL, soapHeader, content);
         } catch (Exception e) {
-            logger.debug("Failed to set binaryState '{}' for device '{}': {}", value, getThing().getUID(),
+            logger.debug("Failed to set timerStart '{}' for device '{}': {}", value, getThing().getUID(),
```

Comment:
Why this change? In line 632 seems to be binary state?

Reply:
Was a copy and paste error before ...